libs/api_common/api_common.hpp

```diff
@@ -264,6 +264,10 @@ ebpf_result_to_errno(ebpf_result_t result)
         error = EEXIST;
         break;
 
+    case EBPF_VERIFICATION_FAILED:
+        error = EACCES;
```

Comment:
should this be documented as possible return value for the new APIs in libbpf.h ?

Reply:
Yes, updated